python/extract_decrypt_kms.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+"""
+@attention: inspired by
+- https://github.com/zalando/kmsclient on 2015 10 15
+- https://github.com/zalando-stups/taupage/blob/master/runtime/opt/taupage/bin/decrypt_kms.py on 2015 10 15
+"""
+
+import argparse
+import yaml
+import boto3
+import base64
+# import pprint
+
+
+class Kms:
+
+    @classmethod
+    def __init__(cls,
+                 file: str = None,
+                 key: str = None,
+                 region: str = None):
+        cls.file = file
+        cls.key = key
+        cls.region = region
+        cls.service_name = 'kms'
+
+    @classmethod
+    def extract_kms_string(cls) -> str:
+        """
+        Prints value of provided key based on given yml file
+        :param arguments: file taupage yml and kms key within yml details
+        :return: the kms string identified by the kms key
+        """
+
+        data = yaml.safe_load(cls.file)
```

Comment:
`cls` will not have a `file` attribute since this is only created on `__init__`. Are you sure this shouldn't be a regular instance method? You should also check if the `file`attribute exists since it can be `None`.